<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Threshold</class>
 <widget class="QWidget" name="Threshold">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Image Type</string>
   </property>
   <widget class="QRadioButton" name="image_color">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>25</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Color</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="image_grey">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>25</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Gray</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="window">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>320</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>window</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>261</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Threshold Type : </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="typeBox">
      <item>
       <property name="text">
        <string>Binary</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Binary inv</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Truncate</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>To zero</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>To zero inv</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>38</y>
     <width>267</width>
     <height>126</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="radioMan">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Manual</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="manualValue">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="radioAdap">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Adaptive</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Mask : </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="mask">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>3</number>
          </property>
          <property name="singleStep">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Constant :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="constant">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <double>-99.000000000000000</double>
          </property>
          <property name="value">
           <double>5.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Method Type :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="methodBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <item>
           <property name="text">
            <string>mean</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>gaussian</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>manualValue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioMan</sender>
   <signal>toggled(bool)</signal>
   <receiver>manualValue</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioAdap</sender>
   <signal>toggled(bool)</signal>
   <receiver>mask</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioAdap</sender>
   <signal>toggled(bool)</signal>
   <receiver>constant</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioAdap</sender>
   <signal>toggled(bool)</signal>
   <receiver>methodBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
